client/js/components/user/DpUserList/DpUserList.vue

```diff
@@ -279,6 +287,16 @@ export default {
         .then(() => {
           this.getItemsByPage()
         })
+    },
+
+    updateAndSaveUser (id) {
```

Comment:
since this is not a generic update-method but sets the delete-flag explicitly, I would name the method accordingly.

Reply:
totally agree :) https://github.com/demos-europe/demosplan-core/pull/4358/commits/94e8271cb8c47ab2922fac1e4a48185e6ae2dea7